client/app/components/ovh-ui-angular/ovh-ui-angular.module.js

```diff
@@ -16,6 +60,33 @@ angular.module("App").run(($q, $translate, $translatePartialLoader, ouiFieldConf
                 pattern: $translate.instant("common_field_error_pattern")
             }
         };
+
+        ouiNavbarConfiguration.translations = {
+            notification: {
+                markRead: $translate.instant("common_navbar_notification_mark_as_read"),
+                markUnread: $translate.instant("common_navbar_notification_mark_as_unread")
+            }
+        };
+
+        ouiPaginationConfiguration.translations = {
+            resultsPerPage: $translate.instant("common_pagination_resultsperpage"),
+            ofNResults: $translate.instant("common_pagination_ofnresults")
+                .replace("TOTAL_ITEMS", "{{totalItems}}"),
```

Comment:
This is the exact same code in cloud, could you explain?

Reply:
translate-values is the common way with angular-translate to replace values in translations. why using string replace instead ?

Reply:
Because this code comes for Web, Cloud and Telecom managers and I just made a copy of it. 

qtlid="452982" id="common_pagination_ofnresults" is a traduction that already exist with this content in fr_FR `sur TOTAL_ITEMS résultats`.
It's been design to accommodate the old and the new translator (using Object or an Array for parameters), I don't want to fork every translations of the UI-KIT so it will stay this way.